pom.xml

```diff
@@ -1225,5 +1225,31 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
+      <profile>
+        <id>owasp-dependency-check</id>
+        <build>
+          <plugins>
+            <plugin>
+              <groupId>org.owasp</groupId>
+              <artifactId>dependency-check-maven</artifactId>
+              <version>6.1.6</version>
```

Comment:
typically  we add the plugin and its version under `<pluginDependencies>` and reference it elsewhere